modules/score_model_to_data.nf

```diff
@@ -16,6 +17,6 @@ process SCORE_MODEL_TO_DATA {
 
     script:
     """
-    status=\$(${scoring_script} '${predictions}' '${results}' '${status}')
+    status=\$(python3 ${scoring_script} '${predictions}' '${goldstandard}' '${results}')
```

Comment:
Does the scoring_script still need to be parameterized? I thought the plan was to have everyone follow the example from the PEGS challenge so that the scripts are always named the same and are located in the same directory in the provided organizer containers?

I think this is potentially an example of an opportunity to just enforce a standard and eliminate parameters that need to be passed to the workflow.

Reply:
I was planning to have `scoring_script` and `validation_script` be the relative paths _to_ the scripts within the users' containers, so that location and name can be flexible.

Reply:
This is OK but IMHO unnecessary unless there is a specific reason that we need to allow scripts to be in different locations in the organizers' containers. If we provide a template container + scripts (such as the `pegs-evaluation` repo) it will be easy for others to follow the example for future challenges.

Reply:
One thing I forgot to mention is I'm planning to add support for non-python validation & scoring scripts, so not all scripts will be called `validate.py` etc.

Reply:
Ah ok. Should that whole part of the command be parameterized then? I.e. 'python3 path/to/validate.py'?

Reply:
I was initially planning to introduce a separate parameter called `params.interpreter` and "build up" the shell execution in `VALIDATE` and `SCORE` that way.

But it's probably easier for the user if there's just 1 parameter to handle the interpreter + the script. I can rename these script parameters to `execute_validation` and `execute_scoring` and format it this way.